ONSdigital/babbage
# Feature/update cookies privacy links

### What

Separated out `Cookies and Privacy` link into a `Cookies` link and a `Privacy` link in the footer. These point to `/cookies` and `/help/privacy` respectively, and are rendered only if the cookies control flag is enabled. If the flag is disabled, it will instead render the `Cookies and Privacy` link.

### How to review

- Check changes by enabling/disabling the `ENABLE_COOKIES_CONTROL` flag in one of the `run` scripts.
- Check that the links direct to the correct routes as per the screenshot in the ticket
- You may need the cookies controller and cookies journey enabled in order to check that the pages load. The initial Cookies and Privacy link should load unless that has moved since.
- Check that the localisation tags are logical

### Who can review

Anyone bar me
